app/main/views.py

```diff
@@ -10,6 +10,11 @@ def index():
     return render_template('main/index.html')
 
 
+@main.route('/map', methods=['GET', 'POST'])
+def mapusers():
```

Comment:
i called it "mapusers" because we're still deciding whether we needed a new route for the resource map -- what do you think I should do instead?